src/cpr_sdk/parser_models.py

```diff
@@ -117,6 +127,22 @@ def to_string(self) -> str:
         return " ".join([line.strip() for line in self.text])
 
 
+class PDFTextBlockV2(TextBlock):
+    """Text block parsed from a PDF document"""
+
+    idx: Annotated[Optional[int], Field(strict=True, ge=0)] = None
+    heading_id: Optional[str] = None
+    bounding_boxes: Optional[list[list[tuple[float, float]]]] = None
+    pages: Optional[list[int]] = None
```

Comment:
thought: `bounding_boxes` and `pages` are related from my understanding in that index n of bounding boxes corresponds to page n of pages. 

Therefore, do we want to model them as a list of tuples - something like?
```python
[(0, [(1.1, 1.1), (2.2, 2.2)]), ...]
```

```python
list[tuple[int, list[tuple[float, float]]]]
```

Reply:
@kdutia: wdyt?